middle_end/flambda2/to_cmm/to_cmm_set_of_closures.ml

```diff
@@ -547,11 +588,17 @@ let let_static_set_of_closures0 env res closure_symbols
       closure_symbol_for_updates
     }
   in
-  let l, free_vars, length, env, res, _effs, updates =
+  let l, memory_chunks, free_vars, length, env, res, _effs, updates =
     Static.fill_layout (Some for_static_sets) decls dbg
       ~startenv:layout.startenv value_slots env res Ece.pure ~prev_updates
       layout.slots
   in
+  (match memory_chunks with
+  | None -> ()
```

Comment:
I think we should consider removing this `option` type: it isn't clear to me what extra safety it brings, and it causes a lot of clutter.

Reply:
Can the list not just be empty in this case?

Reply:
I've removed the option